services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerClusterConfig.java

```
@@ -967,6 +988,14 @@ public VeniceControllerClusterConfig(VeniceProperties props) {
     this.serviceDiscoveryRegistrationRetryMS =
         props.getLong(SERVICE_DISCOVERY_REGISTRATION_RETRY_MS, 30L * Time.MS_PER_SECOND);
     this.pushJobUserErrorCheckpoints = parsePushJobUserErrorCheckpoints(props);
+    this.repushOrchestratorClassName = props.getString(REPUSH_ORCHESTRATOR_CLASS_NAME); // TODO LC: default value?
```

It doesn't have default value. Maybe throw an error if log compaction service is enabled but this configuration is null or missing?

---

discussed offline: if service enabled and this config missing, bring down controller.